classes/task/FileLoader.inc.php

```diff
@@ -126,7 +127,39 @@ function executeActions() {
 		if (!$this->checkFolderStructure()) return false;
 
 		$foundErrors = false;
+		// get setting value
+		$memoryLimit = ini_get('memory_limit');
+		// multiply for shorthand notation
+		switch (strtolower($memoryLimit[strlen($memoryLimit)-1])) {
```

Comment:
I don't think so.  We're just getting the units character for the switch statement here.